apps/code-examples/tsconfig.app.json

```diff
@@ -1,7 +1,9 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
-    "outDir": "../../dist/out-tsc"
+    "outDir": "../../dist/out-tsc",
+    "target": "ES2022",
+    "useDefineForClassFields": false
```

Comment:
In previous migrations I removed `target` and added it to the `tsconfig.base.json` so that we have one source of truth. Might be worth putting the `useDefineForClassFields` property in the base config, too. One benefit of this change is fewer files in the diff!